legion/engine/core/core.vcxproj

```diff
@@ -137,6 +139,15 @@ del /q ".\xcopyexclude"</Command>
     <ClInclude Include="async\ring_sync_lock.hpp" />
     <ClInclude Include="async\thread_util.hpp" />
     <ClInclude Include="async\wait_priority.hpp" />
+    <ClInclude Include="autogen\autogen.hpp" />
+    <ClInclude Include="autogen\autogen_prototype_position.hpp" />
+    <ClInclude Include="autogen\autogen_prototype_rotation.hpp" />
+    <ClInclude Include="autogen\autogen_prototype_scale.hpp" />
+    <ClInclude Include="autogen\autogen_prototype_velocity.hpp" />
+    <ClInclude Include="autogen\autogen_reflector_position.hpp" />
+    <ClInclude Include="autogen\autogen_reflector_rotation.hpp" />
+    <ClInclude Include="autogen\autogen_reflector_scale.hpp" />
+    <ClInclude Include="autogen\autogen_reflector_velocity.hpp" />

```

Comment:
maybe we should only include the autogen files we know for certain will always exist? (aka "autogen\autogen.hpp" and autogen\autogen.cpp)